src/json/nepal.historical.data.json

```diff
@@ -0,0 +1,112 @@
+{
+  "August 15": {
```

Comment:
can we present these days in BS instead? they are static and don't really matter but just for the sake of using BS dates ourselves.

Reply:
yeah done